### DISCUSSION TITLE
sidebar/sanfene/linux

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#273] [Announcements]
# sidebar/sanfene/linux

下载次数超 1 万次，1 万字 1 张手绘图，详解 1 道 Linux 面试高频题（让天下没有难背的八股），面渣背会这些 Linux 八股文，这次吊打面试官，我觉得稳了（手动 dog）。

https://javabetter.cn/sidebar/sanfene/linux.html

**Comment by LVXUEYANGTiger** [comment]
已看

**Comment by echoyolo6** [comment]
已看

**Comment by Kddk-K** [comment]
看了1遍
